### PR TITLE
V1.0.0 beta

### PR DESCRIPTION
**Pull request by wad3g**
### Navigation

Added basic navigation with several configuration options ([#28](https://github.com/KingPixil/wing/issues/28)).

#### Options
* .nav-fixed: navigation will be fixed to the top of the page
* .nav-right: navigation list (links) aligned to the right
* .nav-center: navigation list (links) aligned center

##### Navigation Default
nav-default

##### Navigation Center
nav-center

##### Navigation Right
nav-right

### Editor Config

Added .editorconfig to keep coding style consistent.

### CODE REVIEW

**Comment by KingPixil** [comment]
Thanks for the PR! Love the idea of a nav. I made a couple of changes, making things more appealing, and using system fonts instead of Open Sans. Some files I think that added some extra bloated features were also removed.

Adding a navigation component would be cool, but I think there should be _one_ base class for a navigation, and alignment shouldn't be built in. Why? Wing is meant to be as minimal as possible, using a minimal amount of classes, making it intuitive for developers.

Some improvements:

- No background: Color schemes of sites may vary, and adding a fixed background, while easy to update, should be done by the developer manually.
- If there will be alignment/configuration options, make them more self explanatory:
  - `<div class="nav fixed"></div>`, this could be targeted with something like: `nav.fixed`
- Remove the `color:black`, and leave it to the developer to decide.

Also, if you could `git pull` on your side and update the PR, that would be great 👍

**Comment by wad3g** [comment]
Yeah, I went back and forward between the nav options.

I'll take care of these and send another PR.

File: src/util.css [hunk 1]
```diff
@@ -35,10 +35,26 @@
   justify-content: center;
 }
 
+.right {
```
Comment (KingPixil) [inline src/util.css]: Use something like `.nav.right` instead of `.right`, as it might interfere with other classes.

Comment (wad3g) [inline src/util.css]: `.right` could have multiple uses though. it's the equivalent to `.center`

**Comment by wad3g** [comment]
@KingPixil updated nav utility classes

**Comment by KingPixil** [comment]
Sorry, I see what you mean. Do you think these should be added as utilities, or specifically for the navigation?

**Comment by wad3g** [comment]
I can go either way. 

Personally, nav specific classes for `.nav-right` / `.nav-left` is more intuitive and IMO, but at the same time is someone is building something with flexbox and wanted flexbox utility classes for say `.right` / `.left` the classes would essentially be the exact same.

**Comment by KingPixil** [comment]
Cool, I'd say let's kill two birds with one stone, add `.right`/`.left`, and they can be used for the navigation, and as a utility.

**Comment by wad3g** [comment]
Done. Reverted the utility classes back to their original generic `.right` / `.left` from `.nav.right` / `.nav.left`.

I'm going to hop on your gitter chat - I've got a few questions whenever you get a minute.

**Comment by KingPixil** [comment]
Looking good! I'll be on Gitter for any questions, thanks for the PR! 🙌

**Pull request merged**